EventListener/RateLimitAnnotationListener.php

```diff
@@ -64,7 +67,12 @@ public function onKernelController(FilterControllerEvent $event)
 
         // Find the best match
         $annotations = $event->getRequest()->attributes->get('_x-rate-limit', array());
+
         $rateLimit = $this->findBestMethodMatch($event->getRequest(), $annotations);
```

Comment:
if `findBestMethodMatch` has been deprecated, this library should not use it too

Reply:
Nice catch! 
Indeed, I already moved the logic into class methods, so do not use `findBestMethodMatch` and keep it just for BC